yt_dlp/extractor/abematv.py

```diff
@@ -300,6 +311,11 @@ class AbemaTVIE(AbemaTVBaseIE):
     _TIMETABLE = None
 
     def _perform_login(self, username, password):
+        auth_cache = self.cache.load('abema', 'usertoken')
+        if auth_cache and auth_cache[0] == username and self._get_media_token():
```

Comment:
```suggestion
        if self.cache.load('abema', username) and self._get_media_token():
```
Use username as key so that multiple accounts may be cached